src/components/quote.js

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from 'react';
+
+const apiKey = 'MjFx93DKQY9OcA2CcwGaFg==97fQaXcDcvJR7aL3';
+const apiUrl = 'https://api.api-ninjas.com/v1/quotes';
+
+const Quote = () => {
+  const [data, setData] = useState({ quote: 'this is a quote', author: null });
+
+  const getQuote = async () => {
+    setData({ quote: 'Loading...', author: null });
```

Comment:
- [ ] Good job on showing quotes loading but, Kindly create a separate state for loading and use it instead in order to make the `data` state only responsible for storing quote data.

### Hint
```javascript
const [isLoading, setIsLoading] = useState(false)
```